cli/tests/unit/fetch_test.ts

```diff
@@ -249,6 +249,27 @@ unitTest(
   },
 );
 
+unitTest(
+  { perms: { net: true } },
+  async function fetchMultipartFormBadContentType(): Promise<void> {
+    const response = await fetch(
+      "http://localhost:4545/multipart_form_bad_content_type",
+    );
+    assert(response.body !== null);
+
+    try {
+      const formData = await response.formData();
+      fail(
+        "Response.formData() didn't throw on a incorrect multipart content type",
+      );
+    } catch (e) {
+      assertEquals(e.message, "Invalid form data");
+      assert(e instanceof TypeError);
+    }
```

Comment:
Nitpick: you can use `assertThrowsAsync` instead of manually failing the test

Reply:
Sure, done.